rust/lance-table/src/io/commit/external_manifest.rs

```diff
@@ -60,11 +74,26 @@ pub struct ExternalManifestCommitHandler {
 
 #[async_trait]
 impl CommitHandler for ExternalManifestCommitHandler {
+    async fn resolve_latest_location(
+        &self,
+        base_path: &Path,
+        object_store: &ObjectStore,
+    ) -> std::result::Result<ManifestLocation, Error> {
+        let path = self.resolve_latest_version(base_path, object_store).await?;
+        Ok(ManifestLocation {
+            version: self
+                .resolve_latest_version_id(base_path, object_store)
+                .await?,
+            path,
+            size: None,
+        })
+    }
+
```

Comment:
I've tried not to make a breaking change in `ExternalManifestCommitHandler`. LMK if you object to anything here.